helmfile/values/grafana-user.yaml.gotmpl

```diff
@@ -64,13 +64,14 @@ grafana.ini:
     enabled: true
     client_id: grafana
     client_secret: {{ .Values.grafana.clientSecret }}
-    scopes: profile email openid
+    scopes: profile email openid groups
     auth_url: https://dex.{{ .Values.global.baseDomain }}/auth
     token_url: https://dex.{{ .Values.global.baseDomain }}/token
     api_url: https://dex.{{ .Values.global.baseDomain }}/api
     allowed_domains: {{ join " " .Values.dex.allowedDomains }}
     allow_sign_up: true
     tls_skip_verify_insecure: {{ not .Values.global.verifyTls }}
+    role_attribute_path: contains(groups[*], 'grafana_admin') && 'Admin' || contains(groups[*], 'grafana_editor') && 'Editor' || 'Viewer'
```

Comment:
So if the group claim is as follows
`"groups": ["grafana_admin"]`
the user would be mapped to the `Admin` role in Grafana?

Reply:
yes - and all users would have viewer rights.  Can add a viewer group too if we think its needed.

Reply:
I'm just guessing here so please bear with me, but I would expect that users want to reuse existing groups here and not be forced to use these quite generic group names. For example, there could be a team/group named Vulcan that should have the Editor role and another named Enterprise that should be Admin.

What I'm trying to say is that I think we need to expose this whole line so that users are free to configure it the way they want. Of course we can have this as default, but it must be possible to change it.